is_core/generic_views/form_views.py

```diff
@@ -589,14 +587,16 @@ class AddModelFormView(DefaultCoreModelFormView):
     messages = {'success': _('The %(name)s "%(obj)s" was added successfully.'),
                 'error': _('Please correct the error below.')}
 
+    permission = PermissionsSet(
+        get=CoreCreateAllowed(),
```

Comment:
proč ne CoreReadAllowed()?

Reply:
je to add view a to může být zobrazeno jen když je povoleno vytvořit objekt